### ISSUE TITLE
accuracy

### ISSUE THREAD

**Issue by DreamerDW** [snash4/GAT2VEC#5]
The accuracy is too low to match the accuracy in the paper

**Comment by snash4** [comment]
Hi, 
 What is the accuracy you get on the datasets?

**Comment by DreamerDW** [comment]
I am very honored to receive your reply.
According to your request, ‘python __main__.py --data M10’，when I use GAT2VEC to learn M10, F1 is only 20%，the program does not have any error prompts at run time, just some warnings.
``loading structural graph
/home/GAT2VEC/data/M10/M10_graph.adjlist
Number of nodes: 10310
loading attribute graph
/home/GAT2VEC/data/M10/M10_na.adjlist
Number of nodes: 14296
Random Walks on Structural Graph
------------ATTRIBUTE walk--- 
Learning Representation
Learned Represenation Saved
TR ...  0.1
/usr/local/lib/python2.7/site-packages/sklearn/model_selection/_split.py:1788: FutureWarning: From version 0.21, test_size will always complement train_size unless both are specified.
  FutureWarning)
/usr/local/lib/python2.7/site-packages/sklearn/linear_model/logistic.py:433: FutureWarning: Default solver will be changed to 'lbfgs' in 0.22. Specify a solver to silence this warning.
  FutureWarning)
/usr/local/lib/python2.7/site-packages/sklearn/linear_model/logistic.py:1300: UserWarning: 'n_jobs' > 1 does not have any effect when 'solver' is set to 'liblinear'. Got 'n_jobs' = 8.
  " = {}.".format(effective_n_jobs(self.n_jobs)))
/usr/local/lib/python2.7/site-packages/sklearn/metrics/classification.py:1143: UndefinedMetricWarning: F-score is ill-defined and being set to 0.0 in labels with no predicted samples.
  'precision', 'predicted', average, warn_for)
TR ...  0.3
TR ...  0.5
SVC Training Finished
results
Results .....
  DATASET   TR  accuracy   f1macro   f1micro
0     M10  0.1  0.202134  0.083801  0.202134
1     M10  0.3  0.215228  0.088086  0.215228
2     M10  0.5  0.218720  0.089985  0.218720

**Comment by snash4** [comment]
Thanks for the update.
Could you please try to run it for the other two datasets as well?

**Comment by DreamerDW** [comment]
Thank you very much for your attention to my problems.
I also ran two other datasets and the results were not satisfactory.I'm very interested in GAT2VEC, but I don't know what the problem is.Could you help me find out why?
`  DATASET   TR  accuracy   f1macro   f1micro
0    dblp  0.1  0.521267  0.415245  0.521267
1    dblp  0.3  0.525811  0.422049  0.525811
2    dblp  0.5  0.526436  0.423132  0.526436
`

**Comment by DreamerDW** [comment]
DATASET   TR  accuracy   f1macro   f1micro
0    dblp  0.1  0.521267  0.415245  0.521267
1    dblp  0.3  0.525811  0.422049  0.525811
2    dblp  0.5  0.526436  0.423132  0.526436

**Comment by snash4** [comment]
thanks for your interest. 
Can you check the versions of the required libraries to help me identify the issue?

**Comment by DreamerDW** [comment]
Oh my god！It's working！I reconfigured the versions of the required libraries according to the repuiement.txt.
It's my fault,I apologize for wasting your valuable time.
Thanks again for your reply！

**Comment by snash4** [comment]
It is okay. 
Enjoy working
Any other issue, get back to me again.
